src/data/equiv/algebra.lean

```diff
@@ -190,6 +199,112 @@ protected def discrete_field [discrete_field β] : discrete_field α :=
 end instances
 end equiv
 
+-- Do this definition and two lemmas go somewhere else?
+def is_add_hom {α β : Type*} [has_add α] [has_add β] (f : α → β) : Prop :=
+∀ x y, f (x + y) = f x + f y
+
```

Comment:
Probably `algebra.group` is the best place.

Reply:
If we add this, it has to be glued to `is_add_monoid_hom` and friends.

Reply:
What glue are you proposing Johan? I've moved them.

Reply:
Well, and `is_add_hom` between two `add_groups` is the same thing as an `is_add_group_hom`... Maybe this means we should just remove `is_add_group_hom`? Otherwise there should at least be definitions that allow one to move between the two. However, those can't be instances, because that would lead to loops.
I don't know what's best. I do agree that `is_add_hom` is a useful thing.